src/css/map-view.css

```diff
@@ -339,7 +339,7 @@
   max-height: 100%;
   overflow: auto;
   /* required to be placed above map widget in firefox: */
-  z-index: 1;
+  z-index: 3;
```

Comment:
In my experience this is the kind of thing that is destined to be a regression (especially without screenshot tests). One small thing that has worked for me is defining a variable for the z-index you're interested in, and then doing some math based on that z-index. So if you have 
```
--map-widget-z-index: 1; 
```

Then here you could have
```
z-index: calc(2 + var(--map-widget-z-index);
```

Reply:
^ good tip @ianguerin, I'm going to use that

Reply:
much better!